podcasts/DiscoverViewController+DiscoverDelegate.swift

```diff
@@ -7,7 +7,7 @@ extension DiscoverViewController: DiscoverDelegate {
         if let category {
             reload(except: [item], category: category)
         } else {
-            reloadDiscoverTapped(NSObject())
+            reload(except: [item], category: nil)
```

Comment:
Reload needs to be different than a full reload so we should use the same mechanism as reloading a category.

If we switch this to a `CollectionView` we may be able to rely on the data source diffing to animate more cleanly between states.